viofs/svc/virtiofs.cpp

```diff
@@ -487,6 +483,35 @@ static DWORD FindDeviceInterface(PHANDLE Device)
     return ERROR_SUCCESS;
 }
 
+static DWORD FindDeviceInterface(PHANDLE Device, const std::wstring& Tag)
+{
+    if (Tag.empty())
+    {
+        return FindDeviceInterface(Device, 0);
+    }
+
+    for (DWORD MemberIndex = 0; ; MemberIndex++)
+    {
+        DWORD Error = FindDeviceInterface(Device, MemberIndex);
+        if (Error != ERROR_SUCCESS)
+        {
+            return Error;
+        }
+
+        WCHAR VolumeName[37];
```

Comment:
This array size looks strange. Is this size of something? Can you use some MS define with this value? Or just add comment.

Reply:
This is 36 + 1 where 36 is tag size from VirtIO-FS spec. We have such a definition for the driver. I should be moved to common header file and used here.